src/chapters/sql-part2/complex-queries.rst

```diff
@@ -0,0 +1,350 @@
+Complex Queries
+===============
+
+*Simple* SQL queries are commands that perform straightforward data retrieval,
+usually from only one table at a time. Examples of these queries include using
+the ``SELECT`` and ``WHERE`` keywords, and they narrow the result set by
+focusing on a single parameter.
+
+.. admonition:: Example
+
+   .. sourcecode:: SQL
+      :linenos:
+
+      SELECT *
+      FROM movies
+      WHERE release_date >= 2015;
+
+   Note that in line 1, we selected all of the fields in the ``movies`` table,
+   but we could easily replace ``*`` with a smaller set of column names.
+
+*Complex* SQL queries go beyond the standard requests by retrieving data from
+several tables and by limiting the result set with multiple conditions.
+
+In the :ref:`previous chapter <sql-part1>`, you learned how to use joins to
+display data that is spread over different tables. Another advanced SQL tool is
+called a **subquery**, which is a simple SQL command embedded within another
+query. By nesting queries, you can set up larger restrictions on the data
+included in the result set.
+
+Complex queries often combine joins and subqueries, and they may also include
+multiple ``AND``/``OR`` operators within the ``WHERE`` clause.
+
+Review Joins
+------------
+
+Code along with the following examples. Use the ``writing_supply``,
+``pencil_drawer``, and ``pen_drawer`` tables you created earlier.
```

Comment:
Simplify to "Code along with the following examples using the ``storage`` database you created earlier."

Reply:
Done.  Thanks.